internal/server/scanner.go

```diff
@@ -111,6 +109,19 @@ func (s *Server) newScanner(
 	if sc.col != nil {
 		sc.fmap = sc.col.FieldMap()
 		sc.farr = sc.col.FieldArr()
+		// This fills index value in wheres/whereins
+		// so we don't have to map string field names for each tested object
+		var ok bool
```

Comment:
We could preallocate the where and wherein arrays here so they never have to grow:
```
if len(wheres) > 0 {
    sc.wheres = make([]whereT, 0, len(wheres))
}
```